src/qilib/configuration_helper/adapters/d5a_instrument_adapter.py

```diff
@@ -49,9 +49,10 @@ def __init__(self, address: str, instrument_name: Optional[str] = None) -> None:
     def apply(self, config: PythonJsonStructure) -> None:
         """ Applies configuration
 
-        Applies the configuration update to instrument for dac parameters (step, inter_delay and unit)
-        And, compares all other configuration value with setter command to the existing configuration
-        and raised error in case of mismatch
+        1. Apply configuration update for step, inter_delay
+        2. Apply configuration update for unit parameter based on dac1 unit
```

Comment:
update unit of dac1 values based on... maybe? Since unit is not a qucodes parameter but a unit for the dac parameters